airflow/providers/apache/impala/provider.yaml

```diff
@@ -0,0 +1,41 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+---
+package-name: apache-airflow-providers-apache-impala
+name: Apache Impala
+description: |
+    `Apache Impala <https://impala.apache.org/>`__
+versions:
+  - 2.4.1
+
+dependencies:
+  - impyla>=0.18.0,<1.0
```

Comment:
@potiuk should we consider to write a precommit that verify all providers have apache-airflow in provider yaml dependencies? I'm not sure because it seems like some providers don't have it 
https://github.com/apache/airflow/blob/8f5e100f30764e7b1818a336feaa8bb390cbb327/airflow/providers/http/provider.yaml#L38

Reply:
@blcksrx this still need to be handled..
We need to add minimum Airflow version `apache-airflow>=2.3.0`